extensions/coap/tests/CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ FOREACH(testfile ${CURL_INTEGRATION_TESTS})
   	add_executable("${testfilename}" "${testfile}")
   	target_include_directories(${testfilename} BEFORE PRIVATE ${COAP_INCLUDE_DIRS})
   	target_include_directories(${testfilename} BEFORE PRIVATE ${CURL_INCLUDE_DIRS})
+  	target_include_directories(${testfilename} PRIVATE BEFORE "${CMAKE_SOURCE_DIR}/extensions/standard-processors")
```

Comment:
Why different order of "private" and "before" here?

Reply:
I copied and pasted it. Good catch!